utils/pipeline/pipeline_test.go

```diff
@@ -108,10 +108,18 @@ func TestPipeline(t *testing.T) {
 			}
 		}()
 
+		out := make(chan int)
+		go Bridge[int](ctx, out, chOfInts) // Blocking writes into `out`
```

Comment:
We either need to remove `go` here or inside `Bridge`, otherwise we're creating new goroutine twice.